packages/reports/addon/components/visualization-selector.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *  {{visualization-selector
+ *    report=report
+ *    validVisualizations=validVisualizations
+ *    onVisualizationTypeUpdate=(action 'onVisualizationTypeUpdate')
+ *  }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/visualization-selector';
+import { layout as templateLayout, classNames } from '@ember-decorators/component';
+import { computed } from '@ember/object';
+
+@templateLayout(layout)
+@classNames('visualization-selector')
```

Comment:
```suggestion
@tagName('')
```

and manually add `<div class="visualization-selector" ...attributes>`